tensorflow/python/keras/applications/resnet.py

```diff
@@ -220,7 +219,8 @@ def ResNet(stack_fn,
         BASE_WEIGHTS_PATH + file_name,
         cache_subdir='models',
         file_hash=file_hash)
-    model.load_weights(weights_path)
+    by_name = True if 'resnext' in model_name else False
+    model.load_weights(weights_path by_name=by_name)
```

Comment:
Should this be 
```suggestion
    model.load_weights(weights_path, by_name=by_name)
```

Reply:
Good catch, thanks! I'm still waiting on @tanzhenyu though, we'll see how this goes.